### PR TITLE
Update android/storage.md

### PR DESCRIPTION
**Pull request by kvasukib**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: android/storage.md [hunk 1]
```diff
@@ -501,6 +471,37 @@ TransferObserver observer = transferUtility.upload(
...
+TransferUtilityOptions options = new TransferUtilityOptions();
+options.setTransferThreadPoolSize(8);
+
+TransferUtility transferUtility = TransferUtility.builder()
+    // Pass-in S3Client, Context, AWSConfiguration/DefaultBucket Name
+    .transferUtilityOptions(options)
+    .build();
+```
+
+#### TransferNetworkConnectionType
```
Comment (cbommas) [inline android/storage.md]: Is the TransferServiceCheckTimeInterval no longer required?

File: android/storage.md [hunk 2]
```diff
@@ -501,6 +471,37 @@ TransferObserver observer = transferUtility.upload(
...
+```java
+TransferUtilityOptions options = new TransferUtilityOptions(10, TransferNetworkConnectionType.WIFI);
+
+TransferUtility transferUtility = TransferUtility.builder()
+    // Pass-in S3Client, Context, AWSConfiguration/DefaultBucket Name
+    .transferUtilityOptions(options)
+    .build();
+```
+
+By specifying `TransferNetworkConnectionType.WIFI` you are restricting the data to be transferred to and from S3 only when there is a WiFi connection that is available.
```
Comment (cbommas) [inline android/storage.md]: Change to - By specifying `TransferNetworkConnectionType.WIFI` , data transfers to and from S3 will only happen when the device is on a WiFi connection

File: android/storage.md (hunk 1 above)
Comment (kvasukib) [reply android/storage.md]: Yes. we have removed the feature of the cron job that checks for transfers and marked this API as deprecated. Hence, I am removing it from the docs.

File: android/storage.md (hunk 2 above)
Comment (kvasukib) [reply android/storage.md]: Changed.

**Pull request merged**